troposphere/settings/default.py

```diff
@@ -7,7 +7,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/1.6/ref/settings/
 """
-UI_VERSION = "Zesty Zapdos"
+UI_VERSION = "v28"
```

Comment:
This will (hopefully) be the final iteration. We should internalize this variable and have it set in `clank` via `.j2`...